Fix older bash failing on unbound array variables

huangtianhua opened the pull request:
Fix older bash failing on unbound array variables

Older bash versions fail when trying to expand array variables
when the array is empty.

h00130372 commented:
I have signed the cla

cblecker commented on hack/local-up-cluster.sh:
```diff
@@ -671,7 +671,7 @@ function start_cloud_controller_manager {
       --v="${LOG_LEVEL}" \
       --vmodule="${LOG_SPEC}" \
-      "${node_cidr_args[@]}" \
+      ${node_cidr_args[@]+"${node_cidr_args[@]}"} \
```
```suggestion
      "${node_cidr_args[@]:-}" \
```

cblecker commented:
/ok-to-test

cblecker commented:
@h00130372 The CLA system is still reporting that you haven't signed the CLA. Can you check if you've followed all the steps outlined in https://github.com/kubernetes/kubernetes/pull/76495#issuecomment-482512071?

Commit:
Fix older bash failing on unbound array variables

Older bash versions fail when trying to expand array variables
when the array is empty.

h00130372 commented:
@cblecker, sorry I checked again, and now it's authorized :)

cblecker commented:
/lgtm
/approve
/release-note-none
/priority backlog
/sig testing
/kind cleanup

cblecker commented:
Thanks for the fix 🚀
